config/env_sample.hjson

```diff
@@ -304,5 +304,11 @@
     "COURSES_ENABLED": false,
 
     # Path to the hjson file contains cron queries
-    "CRON_QUERY_FILE": "config/cron_udp.hjson"
+    "CRON_QUERY_FILE": "config/cron_udp.hjson",
+
+    # Change the default Bigquery Project ID
+    "DEFAULT_PROJECT_ID": "udp-umich-prod",
```

Comment:
I would use a placeholder value here, instead of "udp-umich-prod", like 

`"DEFAULT_PROJECT_ID": "<UDP_institution_id>",`

Reply:
I think this was just empty before. This should all work with the service account and the regular data if it's not set and I don't think it will work yet.